setup.py

```diff
@@ -68,7 +68,7 @@
 
 setup(
     name="anonlink",
-    version='0.14.0',
+    version='0.13.2',
```

Comment:
:wilted_flower:  Aw so hard to please. What do you need to see a **Minor** version increment?

Reply:
sorry